README.md

```
@@ -160,10 +160,10 @@ org.juv25d.App
 3. Open:
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)